CMakeLists.txt

```diff
@@ -67,8 +67,15 @@ include(cmake/SetupPackages.cmake)
 include(cmake/SetupRajaConfig.cmake)
 
 # Configure a header file with all the variables we found.
+get_directory_property( RAJA_DEFINITIONS_INITIAL DIRECTORY ${CMAKE_SOURCE_DIR} COMPILE_DEFINITIONS )
+set(RAJA_DEFINITIONS "")
+foreach(definition ${RAJA_DEFINITIONS_INITIAL})
+  set(RAJA_DEFINITIONS "${RAJA_DEFINITIONS} -D${definition}")
+endforeach()
```

Comment:
Okay, so this might actually be unnecessary. We should have moved all the defines into the config.hxx file? If nothing useful is added to the file by doing this then we could probably remove it. All RAJA needs is C++11 right.